corehq/apps/locations/views.py

```diff
@@ -474,9 +474,9 @@ def users_form(self):
         return form
 
     @property
-    def all_products(self):
+    def active_products(self):
         return [(p.product_id, p.name)
-                for p in SQLProduct.by_domain(self.domain)]
+                for p in SQLProduct.objects.filter(domain=self.domain, is_archived=False).all()]
```

Comment:
out of scope, but i wonder if we should add a new manager or classmethod for this. seems like it should almost be the default. cc @esoergel